stix_shifter_modules/msatp/configuration/config.json

```diff
@@ -39,6 +39,18 @@
             "clientSecret": {
                 "type": "password"
             }
+        },
+        "includeAlerts":{
+            "type": "boolean",
+            "default": false
+        },
+        "includeHostOs": {
+            "type": "boolean",
+            "default": false
+        },
+        "includeNetworkInfo": {
+            "type": "boolean",
+            "default": false
```

Comment:
sorry i didn't notice this earlier. is there any reason on why these parameters are under "configuration" object? I think it should be in "connection" object. you can put these inside the nested options object-
```
"connection": {
      "options": {
        "includeAlerts":{
            "type": "boolean",
            "default": false
        },
        "includeHostOs": {
            "type": "boolean",
            "default": false
        },
        "includeNetworkInfo": {
            "type": "boolean",
            "default": false
         }
      }
}
```

Reply:
thanks, I did this.
I also added another option to enable/disable the original_ref property that keeps the original json payload as base64.